core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -602,15 +612,24 @@ MYSQL_TAR=mysql-5.6.10-linux-glibc2.5-x86_64.tar.gz
 MYSQL_JDBC_URL=http://cdn.mysql.com/archives/mysql-connector-java-5.1
 MYSQL_JDBC_TAR=mysql-connector-java-5.1.23.tar.gz
 
-HIVE_MIRROR_URL=https://archive.apache.org/dist/hive/hive-0.13.1
-HIVE_PREFIX=apache-hive-0.13.1-bin
+HIVE_MIRROR_URL=http://archive.cloudera.com/cdh5/cdh/5
+HIVE_PREFIX=hive-1.1.0-cdh5.4.4
 HIVE_TAR=${HIVE_PREFIX}.tar.gz
 
+if [[ "$HBASE_DISTRO" = "APACHE" ]]; then
```

Comment:
hdp has it's own hive, if use apache hive, i can't promise it can work well ,maybe have some compatible problem. btw the hdp hive tar is integrated in HDP-2.3.0.0-centos6-tars-tarball.tar.gz. i can't find a separate tar..maybe someone can give me one hand to give me the download url